src/Components/GameSession.js

```diff
@@ -57,15 +50,42 @@ export default function GameSession() {
   }, [dauberedTiles, moves]);
 
   useEffect(() => {
-    setRandWords(initRandomWords());
+    const randInts = randomGen(24);
+
+    if (gameboardId !== undefined) {
+      const urlWithId = `${process.env.REACT_APP_GAMEBOARD_URI}${gameboardId}`;
+      const apiServer = process.env.REACT_APP_API_SERVER;
+
+      const config = {
+        method: "get",
+        baseURL: apiServer,
+        url: urlWithId,
+      };
+
+      axios(config)
+        .then((res) => res.data)
+        .then((words) => wordProcessor(words, randInts))
+        .then((processedWords) => setRandWords(processedWords))
+        .catch(() => {
+          const words = wordImporter();
```

Comment:
I have suggestion to put this code in a importDefaultWords() function and call it both in the catch block and the else block instead of repeating this code block twice, .i.e
```
function importDefaultWords(){
          const words = wordImporter();
          const processedWords = wordProcessor(words, randInts);
          setRandWords(processedWords);
}
```
```
.catch(() => {
          importDefaultWords();
}
```

Reply:
Updated with commit fixing per suggestion.